CONTRIBUTING.md

```diff
@@ -0,0 +1,4 @@
+# [Read our Wiki](https://github.com/asetalias/Logistics-and-Event-Contributions/wiki/Contributing-to-ALiAS)
```

Comment:
Add this file to another PR?

Reply:
How many PR does man have to open, this is just a singular change. How about this? I will make a commit of it and you don't squash and merge.
Resolving :heart: